metrics/src/test/java/com/expedia/adaptivealerting/metrics/functions/source/GraphiteQueryResultTest.java

```diff
@@ -0,0 +1,31 @@
+package com.expedia.adaptivealerting.metrics.functions.source;
+
+import lombok.val;
+import org.junit.Test;
+import java.util.Iterator;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class GraphiteQueryResultTest {
+
+    @Test
+    public void testgetGraphiteQueryResultFromJson(){
```

Comment:
what if sampleJsonGraphite value is is empty?

Reply:
added tests